plaidml/edsl/edsl.h

```diff
@@ -25,6 +27,8 @@ class TensorDim;
 class TensorIndex;
 class Value;
 
+using pmlc::util::InterpolationMode;
```

Comment:
We can't do this. `pmlc` is not visible to users of the PlaidML API. We need to make a duplicate copy of this enumeration.

Reply:
Removed this dependency and added a copy of the enum in edsl.h.